models/tasks.js

```diff
@@ -355,7 +355,8 @@ const fetchSkillLevelTask = async (userId) => {
  * @returns {Promise<tasks>|Array}
  */
 const fetchSelfTasks = async (username) => {
-  return await fetchUserTasks(username, [], "startedOn", "desc");
+  return await fetchUserTasks(username, []);
+  // removed startedOn as now task is not have field startedOn
```

Comment:
Can you mention the Ticket No here in the comment as well. will be easy to track

Reply:
https://github.com/Real-Dev-Squad/website-backend/issues/1257 & https://github.com/Real-Dev-Squad/website-backend/issues/1256

Reply:
I had mentioned